pvr.nextpvr/resources/language/resource.language.en_gb/strings.po

```diff
@@ -289,3 +289,19 @@ msgstr ""
 msgctxt "#30192"
 msgid "%s is not installed"
 msgstr ""
+
+msgctxt "#30193"
+msgid "Show season folders"
+msgstr ""
+
+msgctxt "#30693"
+msgid "Create separate folders for multiple seasons of recordings"
+msgstr ""
+
+msgctxt "#30194"
+msgid "Use backend genre text"
+msgstr ""
+
+msgctxt "#30694"
+msgid "Kodi default is DVB genre text.  Default required for Kodi genre search and language translation"
+msgstr ""
```

Comment:
One place I see and only a missing new line

Reply:
Fixed